### ISSUE TITLE
Push to Docker Registry broke our build

### ISSUE THREAD

**Issue by 0xdavinchee** [graphprotocol/graph-node#4034]
**Do you want to request a *feature* or report a *bug*?**
Bug.

**What is the current behavior?**
When our CI process runs where we docker-compose graph-node repo to set up a local instance of the subgraph, it breaks.
Here is the specific error:

```
Store error: query execution failed: Subgraph `Qmbj5GncTD5fyFid6V2Gpj8tCzffUatY22eckJQiR9iqzY` has not started syncing yet. Wait for it to ingest a few blocks before querying it: {"response":{"errors":[{"message":"Store error: query execution failed: Subgraph `Qmbj5GncTD5fyFid6V2Gpj8tCzffUatY22eckJQiR9iqzY` has not started syncing yet. Wait for it to ingest a few blocks before querying it"}]
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
https://github.com/superfluid-finance/protocol-monorepo/actions/runs/3190729992/jobs/5206222908
Here is the failed build with the exact same code as the working build below.

**What is the expected behavior?**
https://github.com/superfluid-finance/protocol-monorepo/actions/runs/3189691780/jobs/5203801204
Here is a succesful build with the same code as above.

**Comment by 0xdavinchee** [comment]
hello, we are still running into the same issue with v0.28.1: https://github.com/superfluid-finance/protocol-monorepo/actions/runs/3280227780/jobs/5400752721 - just wanted to give you a heads up

**Comment by neysofu** [comment]
We released `v0.28.2` a few hours after `v0.28.1`, you may want to try that!
